gocd/templates/libs/relay-pops.libsonnet

```
@@ -70,8 +79,8 @@ local generic_pops_stage(region) =
 // of clusters, other regions only deploy to a single cluster.
 {
   stages(region)::
-    if region == 'us' then
-      [us_pops_stage()]
+    if region == 'us' || region == 'de' then
+      [pops_stage()]
```

I think thi function should get a `region` input
```suggestion
      [pops_stage(region)]
```

---

Thanks for catching that, fixed.